src/pkg/clouds/azure/aca/cert.go

```
@@ -167,7 +171,7 @@ func waitForBYODdns(ctx context.Context, hostname, expectedCname, expectedTxt st
 		if !promptShown {
 			term.Printf("Configure DNS records for %s:\n", hostname)
 			term.Printf("  CNAME  %s              ->  %s   (subdomain)\n", hostname, expectedCname)
-			term.Printf("  A      %s              ->  <Container Apps environment IP>   (apex)\n", hostname)
+			term.Printf("  A      %s              ->  %s   (apex)\n", hostname, environmentStaticIP(ctx, envsClient, resourceGroup, envName))
```

golangci-lint (2.12.2)

[error] Error return value of `term.Printf` is not checked

(errcheck)